network/interface.go

```diff
@@ -59,6 +60,12 @@ type Transactions interface {
 	WithPersistency() SubscriberOption
 	// DiscoverServices should be called by the VDR to let the network know it has processed and verified a document (update) for the DID.
 	DiscoverServices(updatedDID did.DID)
+	// DiscoverNodes should be called by the VDR on startup to allow the network to discover Nuts nodes
+	// by looking for NutsComm service types in the DID documents.
+	// It's a weird callback (just like DiscoverServices), but having didstore, which in an internal of VDR,
+	// instantiated on the root level is even weirder.
+	// Note we already (have) to do that with DIDResolver, and one is enough.
+	DiscoverNodes(documentFinder types.DocFinder) error
```

Comment:
must it be called on startup of the vdr or after the vdr started?

Reply:
When the DocumentFinder can be used (so in design on/after VDR started, but works right after `VDR.Configure()`). I can clarify this in the comment, but won't lead to better code. I find this part ugly but don't have another solution..?

Reply:
If the VDR doesn't need to configure the resolvers (no more config var), then the resolver can be configured from the root and passed to other engines. Discover can then be done from network?